src/main/scala/MyAkka.scala

```diff
@@ -1,21 +1,16 @@
-import akka.actor.{Actor, ActorSystem, Props}
-import akka.stream.ActorMaterializer
-
-import akka.stream.alpakka.csv.scaladsl.CsvParsing
-import akka.stream.alpakka.csv.scaladsl.CsvToMap
 import java.nio.file.Paths
-import java.nio.charset.StandardCharsets
-
-import akka.NotUsed
 
-import akka.util.ByteString
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.stream.alpakka.csv.scaladsl.{CsvParsing, CsvToMap}
 import akka.stream.scaladsl._
-import akka.stream.{ActorMaterializer}
+import akka.util.ByteString
 
 
 object Main extends App {
   implicit val system = ActorSystem("MyAkkaActor")
```

Comment:
The ActorSystem keeps the application alive, on purpose and by design.
If you want the app to quit you need to stop the system once ready.

Reply:
Thank you for your awesome support.
The following message appears. 

```
Failure(java.util.NoSuchElementException: key not found: state)

Process finished with exit code 0
```

Reply:
You must be running it with some other data then?

Just works:

```
$ sbt run
[info] Loading settings from idea.sbt ...
[info] Loading global plugins from /Users/ktoso/.sbt/1.0/plugins
[info] Loading project definition from /Users/ktoso/code/akka_agg_csv/project
[info] Loading settings from build.sbt ...
[info] Set current project to Akka CSV Aggregate (in build file:/Users/ktoso/code/akka_agg_csv/)
[info] Running Main
DE,6
HI,4
PR,12
TX,138
MA,24
MD,28
ME,22
IA,64
ID,25
MI,50
UT,20
MN,43
MO,43
IL,87
IN,66
MS,30
MT,16
AK,12
AL,24
VA,53
AR,37
NC,46
ND,17
NE,35
RI,3
AZ,27
NH,6
NJ,28
VT,11
NM,18
FL,68
NV,13
WA,27
NY,100
SC,20
SD,14
WI,22
OH,62
GA,39
OK,32
CA,109
WV,45
WY,4
OR,9
KS,32
CO,30
KY,46
GU,2
PA,102
CT,13
LA,38
TN,36
DC,16
Success(Done)
[success] Total time: 2 s, completed Feb 1, 2018 3:21:44 PM
[warn] The `-` command is deprecated in favor of `onFailure` and will be removed in a later version
[warn] The `-` command is deprecated in favor of `onFailure` and will be removed in a later version
[warn] The `-` command is deprecated in favor of `onFailure` and will be removed in a later version
ktoso @ 月/Users/ktoso/code/akka_agg_csv [master*]
$












```

Reply:
Under 2000 rows
![image](https://user-images.githubusercontent.com/21321689/35664653-533ec560-0766-11e8-9e17-e41fd97bd4de.png)

3748 rows. (now uploaded)
![image](https://user-images.githubusercontent.com/21321689/35664701-7aadb5ca-0766-11e8-8f95-bafa6b4a6293.png)